src/views/Project/index.jsx

```diff
@@ -55,23 +47,19 @@ const tagReposMapping = repositories =>
         search: {
           keywords: ['good-first-bug'],
           products:
-            projects[props.match.params.project].products.filter(
-              product => !componentsFromProduct(product).length
-            ).length > 0
-              ? projects[props.match.params.project].products.filter(
-                  product => !componentsFromProduct(product).length
-                )
+            productsList(projects[props.match.params.project].products).length >
+            0
```

Comment:
When checking for elements in an array, instead of doing `.length > 0`, you can simple do `.length`:

```jsx
productsList(projects[project].products).length ?
  ... :
  ...;
```

This will return true when length > 0. False otherwise.

Next, instead of using `props.match.params.project` anytime we want to access `project`, you can do:

```jsx
options: ({ match: { params: { project } } }) => ({
```

and just use `project` instead.